Sources/App/Controllers/Event/Registration/RegistrationControllerHelper.swift

```diff
@@ -62,9 +62,13 @@ extension RegistrationController {
           regFieldId: fieldId,
           regFieldAnswerId: answerId,
           answerValue: answerValue)
-        
-        try eventRegAnswer.save()
+        if let conn = connection {
+          try eventRegAnswer.makeQuery(conn).save()
```

Comment:
This is need for do it into one solid transaction which we start from `RegistrationController.store()`
The transaction need because we must to save eventReg **before** we store user's answers, but we may get error during this process (error in JSON or wrong some field_id) in this case we have to rollback all changes.